### ISSUE TITLE
[OUDS] [Component] Update of button component (v3.2)

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/Orange-Boosted-Bootstrap#3171]
# Description

Verbatim of @MaximeTonnerre 

```
Release note:
- The specific component “On colored bg” has been split into two distinct components:
- A public version offering traditional management of dark and light modes
- A private version (allowing the core team to nest the component with other components) offering customized mode management with four possible configurations:
- Always in light mode
- Always in dark mode
- Light to dark mode
- Dark to light mode
- Consequently, for the private version, the name of the “Inverted color” variant has been replaced to “Mode control”.
```

# Definition of Ready

_To be defined_

# Definition of Done

_To be defined_

# Resources

See internal Figma

**Comment by B3nz01d** [comment]
Tests should be performed on:
* https://deploy-preview-3178--boosted.netlify.app/orange/docs/0.5/components/buttons/

**Comment by jerome-regnier** [comment]
Review Design Figma :
https://www.figma.com/design/yKtkEY1DOl7YDvmt7jq0QF/-OUDS-Core-Lib--Review?node-id=70114-24251&t=uSmBO8R4dlXxMZ0X-1

Ok Design
